source/includes/_lips_contacts.md

```diff
@@ -11,6 +11,7 @@
 | `create` | POST | /contacts |
 | `update` | PUT | /contact/[id] |
```

Comment:
the `show`, `update` and `destroy` routes are listed with `contact` and should be `contacts`.  Since we are changing this file to be more correct these should be updated.